src/main/webapp/homepage_script.js

```diff
@@ -307,23 +308,42 @@ begins the processes of retrieving the user's location*/
 function getTasksForUserLocation() {
     const script = document.createElement("script");
     script.type = "text/javascript";
-    script.src =  "https://maps.googleapis.com/maps/api/js?key=" + MAPSKEY + "&callback=initialize&language=en";
+    script.src =  "https://maps.googleapis.com/maps/api/js?key=" + MAPSKEY + "&callback=initialize&libraries=places&language=en";
     script.defer = true;
     script.async = true;
     document.head.appendChild(script);
-	
+
     // Once the Maps API script has dynamically loaded it gets the user location,
     // waits until it gets an answer updates the global userLoaction variable and then calls
```

Comment:
Should this comment be moved below, or updated?

Reply:
Yes, I'll update this comment to include the most recent changes to the initialize function. Thanks!